limbo/plugins/oxford.py

```diff
@@ -0,0 +1,59 @@
+"""!define - Returns the oxford dictionary definition of the given term"""
+import os
+import re
+import requests
+
+def define(word):
+    app_id = os.environ.get('OXFORD_APP_ID', '')
```

Comment:
Can you point to an example of how you want this in particular done? I'm looking at the weather plugin as one of the few other plugins that have an API key, and they're in in a similar way